lib/manageiq/messaging/kafka/client.rb

```diff
@@ -88,15 +88,13 @@ def rdkafka_connection_opts(options)
           result = {:"bootstrap.servers" => hosts.join(',')}
           result[:"client.id"] = options[:client_ref] if options[:client_ref]
 
-          result[:"sasl.mechanism"]    = "PLAIN"
+          result[:"sasl.mechanism"]    = !!options[:sasl_mechanism] ? options[:sasl_mechanism] : "PLAIN"
```

Comment:
The `!!` is not needed because truthiness is automatic.  You can also use use a simple `||`

```suggestion
          result[:"sasl.mechanism"]    = options[:sasl_mechanism] || "PLAIN"
```